test/fixtures/plugin-express.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+var TraceLabels = require('../../src/trace-labels.js');
+var shimmer = require('shimmer');
+var semver = require('semver');
+var methods = require('methods').concat('use', 'route', 'param', 'all');
+
+var SUPPORTED_VERSIONS = '4.x';
+
+module.exports = function(version, api) {
+  if (!semver.satisfies(version, SUPPORTED_VERSIONS)) {
```

Comment:
What do you think about flipping paths and `patch/unpatch`? What I mean is instead of accessing the patch function through plugin['']['patch'], we get it with plugin['patch'][''] - so the plugin object has these top-level fields: `versions`, `patch`, `unpatch`.